### ISSUE TITLE
Terms of deposit icons not showing correctly (sometimes)

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/happy-heron#3255]
An accessibility violation was noted as animated icons, these animations are due to a broken fa icon, and they should not be.

We should figure out why they are sometimes showing as broken.  I have not found a pattern, sometimes first page load shows them as broken, refresh fixes them.  Doesn't seem to matter what page you are on.  Could have something to do with the terms being loaded in a turbo frame on demand.

**Broken:**

<img width="798" alt="Screen Shot 2023-07-17 at 1 29 04 PM" src="https://github.com/sul-dlss/happy-heron/assets/47137/566fe64d-1796-4b2d-b0b4-cbdad31e2ec3">

**What it should look like:**

<img width="875" alt="Screen Shot 2023-07-17 at 2 07 22 PM" src="https://github.com/sul-dlss/happy-heron/assets/47137/cc2c368b-85fb-40f0-b316-6433c22b04f6">

FAQ = should be question mark icon
Save = should be disk icon
Print = should be printer icon
